### PR TITLE
Secondary branch

### PR DESCRIPTION
**Pull request by pountzas**
# Added files
- index.html 
- styles.css
- /images/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+                <a href="#">SPACE & COSMOS</a>
+            </nav>
+            <nav class="topnav" >
+                <a href="https://www.nytimes.com/">
+                    <img src="./images/New-York-Times-Logo.png" style="width:200px"  alt="NYT logo">
+                </a>
+            </nav>
+            <nav class="toprightnav">
+                <button class="button" type="button">SUBSCRIBE NOW</button>
+                <button class="button" type="button">LOG IN</button>
```
Comment (GzimAsani) [inline index.html]: Please change the color of these buttons and make them look as the original

File: index.html [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+        <section>
+            <div class="h1-hr">
+                <h1 >Space Ripples Reveal Big Bang’s Smoking Gun</h1>
+                <hr>
+            </div>
+            
+            <figure>
+                <img src="./images/18cosmos-superJumbo.jpg" alt="Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.Credit...">
+                <div class="guth-text">
+                    <span>Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</span>
```
Comment (GzimAsani) [inline index.html:51]: This line is extending more than the picture, kindly fix it

File: index.html [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+                <p>He was trying to understand why there was no trace of some exotic particles that should have been created in the Big Bang. Instead he discovered what might have made the universe bang to begin with. A potential hitch in the presumed course of cosmic evolution could have infused space itself with a special energy that exerted a repulsive force, causing the universe to swell faster than the speed of light for a prodigiously violent instant.</p>
+                <p>If true, the rapid engorgement would solve paradoxes like why the heavens look uniform from pole to pole and not like a jagged, warped mess. The enormous ballooning would iron out all the wrinkles and irregularities. Those particles were not missing, but would be diluted beyond detection, like spit in the ocean.</p>
+                <p>“SPECTACULAR REALIZATION,” Dr. Guth wrote across the top of the page and drew a double box around it.</p>
+
+                
+
+            </section>
+
+            <aside>
+                <h4>Editors’ Picks</h4>
```
Comment (GzimAsani) [inline index.html:73]: This section is extending more than it should in to the right side which is making your site to expand more than it should, kindly fix this.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:
Hello **Team** 👋 Good job implementing so far :muscle:, however there are some more changes that you have to implement

Required changes:
- [ ] Add a more detailed description in your pull request
- [ ] Add social media icons which are between title and the main picture 
- [ ] Add the comments button and social media icons at the bottom of the page, this should be after  the line div in line 205
- [ ] Add the `More in Space and Astronomy` and `Most Popular` sections 
- [ ] You are using repated css code like _display: flex_ 4 times and float 5 times this is bad CSS practice what you can do is create a class like `.grid { display: grid; }` and use it every time you need
- [ ] Kindly check the comments below

# You got this :muscle: 

### **Please make the required changes and submit another review request when you are ready.**

<hr>
### if you have any question kindly reach me on slack @GzimAsani

### HAPPY CODING :computer:

File: index.html [hunk 4]
```diff
@@ -0,0 +1,402 @@
...
+    <link rel="stylesheet" href="styles.css">
+    <title>Space Ripples Reveal Big Bang’s Smoking Gun</title>
+    
+</head>
+<body>
+    <header>
+        <div id="outer-nav">
+            <nav class="topleftnav flex float-left">
+                <img src="./images/menu.png" style="width:20px" alt="menu">
+                <img src="./images/magnifier.png" style="width:20px" alt="search">
```
Comment (umairarshadbutt) [inline index.html]: Remove inline Styling

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi Team
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed.

##

Kindly work on the suggested # changes 👇. 
- [ ] Add a more detailed description in your pull request.
- [ ] Add the More in Space and Astronomy and Most Popular sections
- [ ] Remove inline Styling
- [ ] Please add MIT license in the README file

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @pountzas :wave:

__You've done excellently well so far. Your project is looking ready for approval :wink: 
However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] You've highlighted how to clone the project in the Setup section of the Readme, please complete this section by pointing out how to run the project on the browser to your users. It is always better to have a complete Readme without any assumption that the users know how to find their way.
- [ ] The `Issues page` and the `MIT license` links provided on the readme are broken. Please ensure that the MIT license file is present and the links are working.
- [x] Kindly check the image below. I believe the date is in the wrong place.
date

- [x] Your `Most Popular` section is a bit different from the one on the original page. Please check the table below and make necessary adjustments: take note of the spacings and alignments.

| Your Page | Original Page |
| -------------- | ------------------ |
| clone | original |

**[ OPTIONAL CHANGES ]**

- [x] For subsequent commits, try to be more professional with your commit messages. I believe using messages like `code review fix 7th try test1` is not the best. Check out [this article](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) on how to write better commit messages.


* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by SpaYco — APPROVED** [review]
# Project Approved

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**